test/index.test.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import assert from 'assert';
+import Render from '../app/render';
+import { describe, it, before } from 'mocha';
+
+describe('Vue unit test', () => {
+  before(() => {
+    document.body.innerHTML = window.__html__['fixtures/test.html'];
```

Comment:
use `fixture.load`, and `fixture.cleanup`
https://github.com/billtrik/karma-fixture#usage